### PR TITLE
chore/remove deprecated and add missing translateables

### PR DESCRIPTION
**Pull request by Magnus-Kuhn**

### CODE REVIEW

File: packages/runtime/src/dataViews/DataViewTranslateable.ts [hunk 1]
```diff
@@ -7,31 +7,18 @@ export class DataViewTranslateable {
...
-            attributesShareRequestNamePlural: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.namePlural`,
-            // You don't have a relationship to any of the recipients of this request
-            attributesShareRequestNoRelationship: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.noRelationship`,
-            // The data is only shared with known recipients
-            attributesShareRequestOnlyRelationships: `${DataViewTranslateable.prefix}requests.AttributesShareRequest.onlyRelationships`,
-            // Request for changing an attribute
-            attributesChangeRequestName: `${DataViewTranslateable.prefix}requests.AttributesChangeRequest.name`,
-            // Request for changing multiple attributes
-            attributesChangeRequestNamePlural: `${DataViewTranslateable.prefix}requests.AttributesChangeRequest.namePlural`
-        },
```
Comment (jkoenig134) [inline packages/runtime/src/dataViews/DataViewTranslateable.ts:34]: why is this removed?

File: packages/runtime/src/dataViews/DataViewExpander.ts [hunk 2]
```diff
@@ -1742,6 +1742,11 @@ export class DataViewExpander {
             statusText = DataViewTranslateable.transport.relationshipRevoked;
         } else if (relationship.status === RelationshipStatus.Active) {
             statusText = DataViewTranslateable.transport.relationshipActive;
+        } else if (relationship.status === RelationshipStatus.Terminated) {
+            statusText = DataViewTranslateable.transport.relationshipTerminated;
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (jkoenig134) [inline packages/runtime/src/dataViews/DataViewExpander.ts:1747]: remove this disable and fix it properly please.

File: packages/runtime/src/dataViews/DataViewTranslateable.ts (hunk 1 above)
Comment (Magnus-Kuhn) [reply packages/runtime/src/dataViews/DataViewTranslateable.ts:34]: From the Announcing enmeshed v2 requests blog: The `RequestMail`, `AttributesChangeRequest` and `AttributesShareRequest` types are now deprecated and will be removed in the future.

I just went ahead with it.

Comment (jkoenig134) [reply packages/runtime/src/dataViews/DataViewTranslateable.ts:34]: oh, I see!

**Review by jkoenig134 — APPROVED** [review]
